src/util/__tests__/Recoil_deepFreezeValue-test.js

```diff
@@ -28,5 +28,8 @@ describe('deepFreezeValue', () => {
       deepFreezeValue({test: new DataView(new ArrayBuffer(16), 0)}),
     ).not.toThrow();
   });
+  test('check no error: object with Window property', () => {
+    expect(() => deepFreezeValue({test: window})).not.toThrow();
```

Comment:
This also can't access the global `window` directly for SSR compatibility.  We hope to be able to run the test suite in that environment as well one day.  So, need to guard against it with the check described above.